### PR TITLE
main: Added a call to GarbageCollect

### PR DESCRIPTION
**Pull request by whygyc**
#### Which issue(s) this PR fixes:

Fixes: #6981

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: cmd/crio/main.go [hunk 1]
```diff
@@ -342,6 +342,13 @@ func main() {
 			}
 		}
 
+		// We always use "Volatile: true" to create containers, which means that we might
```
Comment (haircommander) [inline cmd/crio/main.go:345]: I don't believe `Volatile: true` is a default value. can this be reworded to 
```
In situations where "Volatile: true" is specified in the container storage, there are containers and layers
that may be lost track of in an unclean shutdown. 
We need to call the garbage collection function to clean up the redundant files.
```

**Comment by kolyshkin** [comment]
@whygyc a nit -- your added code has trailing whitespaces, which trigger warnings from two different linters; please fix this.

**Comment by whygyc** [comment]
> @whygyc a nit -- your added code has trailing whitespaces, which trigger warnings from two different linters; please fix this.

Thanks for your reminding, I have modified it

File: cmd/crio/main.go (hunk 1 above)
Comment (whygyc) [reply cmd/crio/main.go]: I have modified it according to your requirement. But what I can see from the code is that `Volatile: true` is a fixed value and always true.
https://github.com/cri-o/cri-o/blob/e46a5207f2a05ad224b98dfc49f77fb996bb30ce/internal/storage/runtime.go#L260-L267

Comment (haircommander) [reply cmd/crio/main.go]: ah I had the wrong option in mind (I was thinking they were for a transient store, not volatile containers)
you are correct, we are always setting volatile, I think you can revert to what it was before. sorry!

Comment (whygyc) [reply cmd/crio/main.go]: > ah I had the wrong option in mind (I was thinking they were for a transient store, not volatile containers) you are correct, we are always setting volatile, I think you can revert to what it was before. sorry!

That's not a problem. I'm not very confident in my English. 
I trust your judgment and I will revert it if you think it was better before. It's easy and doesn't take much time.

**Review by sohankunkerkar — APPROVED** [review]
/lgtm

**Review by saschagrunert — APPROVED** [review]

**Pull request merged**